www/script.js

```diff
@@ -0,0 +1,186 @@
+mapping = {};
+mapping["sinkhole"] = "c1";
+mapping["destruction"] = "c2";
```

Comment:
I think this would violate separation of concerns, since SVG should really just control presentation. However I plan to move this to ocf-etc so it's not hardcoded in a JS file.